### PR TITLE
Make the polling interval configurable, by default 24h

### PR DESCRIPTION
**Pull request by Wissididom**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,7 +31,54 @@ if (!broadcaster.data) {
 }
 const broadcasterId = broadcaster.data[0].id;
 const broadcasterDisplayName = broadcaster.data[0].display_name;
+const pollingInterval = process.env.POLLING_INTERVAL ?? "1d";
 let date = new Date();
+switch (pollingInterval.substring(pollingInterval.length - 1)) {
+  case "d": // days
+    date.setDate(
+      date.getDate() -
+        +pollingInterval.substring(0, pollingInterval.length - 1),
```
Comment (NiceWaffel) [inline index.js:40]: Maybe use parseInt instead of unary plus.

**Comment by NiceWaffel** [comment]
Current implementation is probably broken. For example subtracting 24h does not change anything about the hour but should subtract 1 from the day (on an normal, non daylight-savings day). Idk if javascript does that. Needs testing.

**Comment by Wissididom** [comment]
LGTM :+1: Seems to work as expected, will test with the real script tomorrow.
image

File: index.js [hunk 2]
```diff
@@ -31,7 +31,54 @@ if (!broadcaster.data) {
 }
 const broadcasterId = broadcaster.data[0].id;
 const broadcasterDisplayName = broadcaster.data[0].display_name;
+const pollingInterval = process.env.POLLING_INTERVAL ?? "1d";
 let date = new Date();
+switch (pollingInterval.substring(pollingInterval.length - 1)) {
+  case "d": // days
+    date.setDate(
+      date.getDate() -
+        parseInt(pollingInterval.substring(0, pollingInterval.length - 1)),
```
Comment (NiceWaffel) [inline index.js:40]: Instead of duplicating the parseInt() call 6 times you could instead put it in a variable.

**Comment by Wissididom** [comment]
Works as expected! :+1:

**Pull request merged**